core/domain/collection_domain.py

```diff
@@ -1043,11 +974,6 @@ def validate(self, strict=True):
                     'Expected to have at least 1 exploration in the '
                     'collection.')
 
-            # Ensure the collection may be started.
-            if not self.first_exploration_id:
```

Comment:
Why do we want to remove this?

Reply:
This check is already done in the previous one [here](https://github.com/oppia/oppia/blob/develop/core/domain/collection_domain.py#L1041), i.e `if not self.nodes`. The presence of self.nodes would itself mean the presence of an exploration.

Reply:
okay, thanks!